### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by gbanas**

### CODE REVIEW

File: setup.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS responses (
+  id    SERIAL PRIMARY KEY,
+  added_date    timestamp WITH time zone,
+  survey_response    jsonb
+);
```
Comment (necrophonic) [inline setup.sql:5]: Should we be creating indexes on the json here to speed searches?

Comment (pipmurphy) [reply setup.sql:5]: Depends what the retrieval patterns are - Postgres will automatically create an index on the primary key, so other than retrieval by "id", what queries do we use?

Comment (gbanas) [reply setup.sql:5]: This is a sample document:

``` javascript
{
  "type": "uk.gov.ons.edc.eq:surveyresponse",
  "origin": "uk.gov.ons.edc.eq",
  "survey_id": "194825",
  "version": "0.0.1",
  "collection": {
    "exercise_sid": "hfjdskf",
    "instrument_id": "10",
    "period": "0616"
  },
  "submitted_at": "2016-03-12T10:39:40Z",
  "metadata": {
    "user_id": "789473423",
    "ru_ref": "1234570071A"
  },
  "data": {
    "1": "2",
    "2": "4",
    "3": "2",
    "4": "Y"
  }
}
```

At the moment we support querying:
- "id" (primary key) and "added_date" columns,
- "survey_id", "survey_response.metadata.ru_ref" and "survey_response.collection.period" fields,

We can also query "survey_response.form" field but I haven't seen this field in test data.

Comment (necrophonic) [reply setup.sql:5]: Can we add in indexes for the keys we're querying then?

Comment (gbanas) [reply setup.sql:5]: I've added indexes in https://github.com/ONSdigital/sdx-store/commit/26453fde416ea6e99171504f64f5ca4448057e85

**Review by necrophonic — APPROVED** [review]
Looks good

**Comment by necrophonic** [comment]
Branch to stay dormant as POC for now. Will need new work to Postgres things when we're ready to move to CF anyhow.

**Pull request closed without merging**